internal/tofu/context_plan.go

```diff
@@ -568,32 +563,100 @@ func (c *Context) postPlanValidateImports(importResolver *ImportResolver, allIns
 func (c *Context) findImportTargets(config *configs.Config, priorState *states.State) []*ImportTarget {
 	var importTargets []*ImportTarget
 	for _, ic := range config.Module.Import {
-		if priorState.ResourceInstance(ic.To) == nil {
-			importTargets = append(importTargets, &ImportTarget{
-				Config: ic,
-			})
-		}
+		importTargets = append(importTargets, &ImportTarget{
```

Comment:
If `priorState` is an unused variable, then we can remove it.

Reply:
Thanks. Fixed [here](https://github.com/opentofu/opentofu/pull/1270/commits/d06b19f31fd33929e4ce754a4dcc99d78a5a242a)